changelog/12535.doc.rst

```diff
@@ -0,0 +1,4 @@
+`This
```

Comment:
I don't think we need a changelog entry for this small change (thanks for writing one anyway!).

I would have deleted the file myself but I don't have access to your repository.